staging/src/k8s.io/apiserver/pkg/admission/plugin/cel/filter.go

```diff
@@ -286,6 +294,32 @@ func CreateAdmissionRequest(attr admission.Attributes) *admissionv1.AdmissionReq
 	}
 }
 
+func CreateNamespaceObject(namespace *v1.Namespace) *v1.Namespace {
+	if namespace == nil {
```

Comment:
This behavior is actually important because that happens for cluster-scoped resource. Let's add the doc for this function before we forget.